### PR TITLE
added end to end test for creating assignment

### PR DESCRIPTION
**Pull request by yoelazauderer**
Hi,
When I run the new test, I am having an error where it says no database is selected - do you know why that could be?

### CODE REVIEW

File: cst438_gradebook/src/test/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# JAWSDB database properties for register back end
+spring.datasource.url = jdbc:mysql://tvcpw8tpu4jvgnnq.cbetxkdyhwsb.us-east-1.rds.amazonaws.com:3306/?user=mjy97vcxj6kkxoyp
```
Comment (josevh) [inline cst438_gradebook/src/test/resources/application.properties:2]: There's no database chosen in the URL.

Comment (yoelazauderer) [reply cst438_gradebook/src/test/resources/application.properties:2]: Oh, thanks - I see I had the wrong last part of the URL. I fixed it and it's working now.
The test started the browser came up and entered the input fields, but is getting stuck when I try to verify that the new assignment shows in the list. The assignment name element doesn't have a data-field name or value so I'm not sure how I should compare it. I guess I will ask the professor on Thursday.

Thank you!

Comment (josevh) [reply cst438_gradebook/src/test/resources/application.properties:2]: Using xpath you can search for elements with a given text content. I believe the (existing) other end-to-end test in the repo (at least on the repo I used) has an example of this.

Comment (yoelazauderer) [reply cst438_gradebook/src/test/resources/application.properties:2]: Thanks I went about accessing that value in a different way and now the test works - thanks!

**Pull request merged**